File: python/Programmers_level/level1/이상한 문자 만들기.py
# 아스키코드 이용
def solution(s):
    l = list(s.split(' '))
    print(l)
    for i in range(len(l)):
        temp = list(l[i])
        for j in range(len(temp)):
            if j == 0 or j % 2 == 0:
                if 96 < ord(temp[j]) < 123: # 소문자인 경우
                    temp[j] = chr(ord(temp[j]) - 32)
            else:
                if 64 < ord(temp[j]) < 91: # 대문자인 경우
                    temp[j] = chr(ord(temp[j]) + 32)
        l[i] = ''.join(temp)
                                        
    answer = ' '.join(l)
    return answer

File: Javascript/Programmers_level/level1/이상한 문자 만들기.js
function solution(s) {
    let answer = [];
    const arr = s.split(" ")
    arr.forEach((item) => {
        let temp = ''
        for (let i = 0; i < item.length; i++) {
            if (i == 0 || i % 2 == 0) temp += item[i].toUpperCase()
            else temp += item[i].toLowerCase()
        }
        answer.push(temp)
    })
    return answer.join(" ");
}